core/templates/dev/head/components/skill-mastery/skill-mastery.directive.ts

```diff
@@ -46,8 +49,32 @@ angular.module('oppia').directive('skillMasteryViewer', [
             return Math.round(ctrl.skillMasteryDegree * 100);
           };
 
+          ctrl.getMasteryChangePercentage = function() {
+            if (ctrl.masteryChange >= 0) {
+              return '+' + Math.round(ctrl.masteryChange * 100);
+            } else {
+              return Math.round(ctrl.masteryChange * 100);
+            }
+          };
+
           ctrl.getLearningTips = function() {
-            // TODO(sophiewu6): Implement learning tips feature.
+            if (ctrl.skillMasteryDegree >= GOOD_MASTERY_CUTOFF) {
+              if (ctrl.masteryChange > 0) {
+                return 'You have mastered this skill very well! ' +
+                  'You can work on other skills or learn new skills.';
+              } else {
+                return 'Seems like you didn’t do very well this time. ' +
```

Comment:
How about changing this text to 
"Looks like your mastery in <skill> dropped. Here are some ways you can improve your mastery:

- Look at the concept card (maybe add a link?)

- Run a practice session
"
@seanlip

Reply:
Hm, perhaps: "Looks like your mastery of this skill has dropped. To improve it, try reviewing the [concept card](...) and then practicing more questions for the skill!"

Reply:
Done. I added a button to open a concept card modal instead of adding a link.